src/ItemBaseV0.sol

```diff
@@ -0,0 +1,209 @@
+pragma solidity ^0.6.7;
+
+// import "zeppelin-solidity/introspection/SupportsInterfaceWithLookup.sol";
+import "zeppelin-solidity/proxy/Initializable.sol";
+import "ds-auth/auth.sol";
+import "./interfaces/ISettingsRegistry.sol";
+import "./interfaces/IObjectOwnership.sol";
+import "./FurnaceSettingIds.sol";
+
+contract ItemBase is Initializable, DSAuth, FurnaceSettingIds {
```

Comment:
Since we have not deploy any ItemBase on mainet yet, we may not use version in names for identification.